streamer/src/nonblocking/testing_utilities.rs

```diff
@@ -239,3 +245,46 @@ pub async fn make_client_endpoint(
         .await
         .expect("Test server should be already listening on 'localhost'")
 }
+
+pub async fn check_multiple_streams(
```

Comment:
This is moved from nonblocking::quic::test to here to be reused for other testing